psykorebase/cli.py

```diff
@@ -62,11 +88,14 @@ def __init__(self, repository_path, primary_branch, secondary_branch=None):
 
     def rebase(self):
         """Actually rebase."""
-        self.checkout(self.primary_branch)
-        self.checkout(self.target_branch, create=True)
-        self.merge(self.secondary_branch,
-                   message='Psycho-rebased branch %s on top of %s'
-                           % (self.secondary_branch, self.primary_branch))
+        if not self.rebase_continue:
+            self.checkout(self.primary_branch)
+            self.checkout(self.target_branch, create=True)
+            self.merge(self.secondary_branch, no_ff=self.no_ff,
```

Comment:
Oh, I did it here: https://github.com/benoitbryon/psykorebase/pull/2/files#diff-b9ec79747ca0a28f0277bd2684ba447eR121